### ISSUE TITLE
ResizeObserver in Safari and IE

### ISSUE THREAD

**Issue by ShaneHudson** [airbnb/visx#985]
I have not got a solid reduced test care to explain this issue, in basic examples it did seem to work fine... but figured it is worth mentioning so you are aware of it.

When using `ParentSize` within a page (including Grid and other bits of layout) I found that Chrome and Firefox worked very well but Safari (as well as requiring a `height`, not just `max-height`) and IE11 both had issues with `ParentSize`. I found that using `ParentSize` in Safari led to a large amount of `loop completed with undelivered notifications` errors, and didn't seem to work at all in IE11.

When I looked further into it I learned about the newer `ParentSizeModern` which appeared to fix Safari's issue but didn't touch IE11.

Long story short, my workaround for the issue was to pull in `resize-observer-polyfill` manually (instead of relying on it via visx) and add the polyfill to the global `window` (which is not ideal). With this done, `ParentSizeModern` worked in all browsers so will be using that for now but I would have expected `ParentSize` to work without this workaround?

**Comment by ShaneHudson** [comment]
Two other issues with IE11:
- DOM parentNode append needs polyfill
- Tooltip has `NaN` for positioning (`top` and `left`) so doesn't work at all. Didn't find a fix for this so for now have removed tooltips for IE.

**Comment by hshoff** [comment]
Hi @ShaneHudson, thanks for checking out visx!

`<ParentSize>` comes with a ResizeObserver polyfill out of the box. For cases like this, we recommend the approach you took where you can use `<ParentSizeModern>` which doesn't include the polyfill, but provides the flexibility to include one of your choosing. We recommend most folks use `<ParentSizeModern>` these days and only include a polyfill when necessary. 

As for the tooltip issue:
> Tooltip has NaN for positioning (top and left) so doesn't work at all. Didn't find a fix for this so for now have removed tooltips for IE.

Surprised to hear this. Could you provide a codesandbox reproducing the issue? This will help us locate the problem.

**Comment by hshoff** [comment]
Closing this as [IE11 end of support](https://docs.microsoft.com/en-us/lifecycle/products/internet-explorer-11) is June 15th, 2022.
